.github/workflows/ci.yml

```diff
@@ -292,3 +389,27 @@ jobs:
           name: playwright-traces-${{ github.run_id }}-${{ github.run_attempt }}-${{ matrix.test-name }}
           path: integration/test-results
           retention-days: 1
+
+  pkg-pr-new:
+    name: Publish with pkg-pr-new
+    if: ${{ github.event_name != 'pull_request' || github.event.pull_request.draft == false }}
+    needs: [check-permissions, build-packages]
+    runs-on: 'blacksmith-8vcpu-ubuntu-2204'
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          show-progress: false
+
+      - name: Setup Node
```

Comment:
If we want turbo caching to work, we'll need these environment variables https://github.com/clerk/javascript/blob/e367ffb03fb03eeaf644b63a51ff9d8642923688/.github/workflows/ci.yml#L113-L116

Reply:
Addressed